composer.json

```diff
@@ -3,11 +3,17 @@
   "type": "drupal-module",
   "description": "Provides Commerce integration for Enhanced Ecommerce events via Google Tag Manager",
   "homepage": "https://www.drupal.org/project/commerce_google_tag_manager",
-  "license": "GPL-2.0+",
+  "license": "GPL-2.0-or-later",
   "keywords": ["drupal", "commerce", "google tag manager", "enhanced ecommerce"],
+  "support": {
+    "issues": "https://www.drupal.org/project/issues/commerce_google_tag_manager"
+  },
   "require": {
-    "drupal/core": "^8.5",
-    "drupal/commerce": "~2",
+    "drupal/commerce": "^2.8",
     "drupal/google_tag": "^1.1"
+  },
+  "require-dev": {
+    "squizlabs/php_codesniffer": "^2.7",
```

Comment:
Thoses will be used later on the Code Quality assertions PR.

Reply:
I will add a post-commit script that anybody will be able to use to asserts the code is compliante with Drupal Code Quality - To prevent the https://pareview.sh/ to be sad

Reply:
@WengerK I just enabled the module on StyleCI.